src/Main.cpp

```diff
@@ -116,49 +117,64 @@ static void SurroundSelectionsWith(char ch1, char ch2) {
 	editor.EndUndoAction();
 }
 
+static char GetASCIICharacterFromKey(WPARAM wparam, LPARAM lparam) 
+{
+	BYTE keyboardState[256];
+	GetKeyboardState(keyboardState);
+
+	// https://msdn.microsoft.com/en-us/library/ms644984(v=VS.85).aspx
+	UINT scanCode = HIBYTE(LOWORD(lparam)); 
+	UINT virtualKeyCode = UINT(wparam);
+	UINT isMenuActive = 0;
+
+	WORD charResult = 0;
+
+	int isDeadKey = ToAscii(
```

Comment:
Why declare a variable and assign the return value when it's not used?

Reply:
I intended to use the variable name to document the used function. That function is essential for the new method of character comparison. Although its return value is not used, the reader can understand it fully without looking it up in the Windows API documentation.